crates/holochain/tests/multi_conductor/mod.rs

```diff
@@ -112,7 +112,11 @@ async fn multi_conductor() -> anyhow::Result<()> {
 
     // See if we can fetch metric data from bobbo
     let metrics = conductors[1].dump_network_metrics(None).await?;
-    println!("@!@! - metrics: {}", metrics);
+    println!("@!@! - metrics: {metrics}");
```

Comment:
Prefer `tracing::info` in tests so it can be turned off by default?

Reply:
Done.